content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -43,20 +44,26 @@ including the `--oauth-no-pkce` flag when starting `chronograf`.
   - `EXPLAIN`
   - `EXPLAIN ANALYZE`
 - Flux improvements and additional functionality:
-  - Add dashboard template variables to Flux query execution.
+  - Add predefined and custom dashboard template variables to Flux query execution.
     Flux queries include a `v` record with a key value pair for each variable.
-  - Support dashboard template variables defined with Flux.
-  - Add Kapacitor Flux Tasks on the Manage Tasks page (read only).
+  - [Define template variables with Flux](/chronograf/v1.9/guides/dashboard-template-variables/#flux-query).
+  - Add [Kapacitor Flux tasks](/chronograf/v1.9/guides/advanced-kapacitor/#manage-kapacitor-flux-tasks)
+    on the Manage Tasks page (read only).
   - Provide documentation link when Flux is not enabled in InfluxDB 1.8+.
-  - Write to buckets when the Flux tab is selected.
+  - [Write to buckets using the Flux `to()` function](/chronograf/v1.9/guides/write-to-influxdb/#use-the-flux-to-function-in-a-query).
```

Comment:
There is a Write UI that allows to insert data from a file or using a line protocol text. Before 1.9, it only allowed to write to databases (v1 API), now it also allows to write to buckets (v2 API). It writes buckets when the Flux mode is selected ... as one might expect since the Flux UI works with buckets.

```suggestion
  - Write to buckets when Flux mode is selected.
```